tests/set-labels/both-additionalLabelFields-fieldSpecs/.expected/results.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha2
+kind: FunctionResultList
+metadata:
+  name: fnresults
+exitCode: 1
+items:
+  - image: gcr.io/kpt-fn/set-labels:unstable
+    stderr: '[error] /// : failed to configure function: fieldSpecs has been deprecated, please rename it to additionalLabelFields'
```

Comment:
where are we with this `////` issue?

Reply:
It has been fixed in https://github.com/kubernetes-sigs/kustomize/pull/3955.
We need to bump kyaml version in functions.

Reply:
https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/382